.github/actions/cache/action.yml

```diff
@@ -1,12 +1,15 @@
 name: 'Cache for all OSes'
 description: 'Needs to be called before any call to ext/.cmd|sh or cmake'
 inputs:
-  use-rust:
-    description: 'Whether rust is used'
-    default: false
   extra-key:
     description: 'Extra cache key to use in the cache name. Useful when several caches are used in one workflow.'
     default: ''
+  use-meson:
+    description: 'Whether meson is used'
+    default: false
+  use-rust:
+    description: 'Whether rust is used'
+    default: false
```

Comment:
Could you check .github/workflows/ci-mingw.yml? It also uses `.github/actions/cache`:

```
    - uses: ./.github/actions/cache
      with:
        extra-key: ${{ matrix.sys }}
```

Note: I used this `find` command to find the files that use `.github/actions/cache`:

```
find .github -name "*" -a -type f | xargs grep github/actions/cache
```